keps/693-multikueue/README.md

```diff
@@ -304,6 +336,23 @@ type MultiKueue struct {
 	GCInterval *metav1.Duration `json:"gcInterval"`
 	Origin *string `json:"origin,omitempty"`
 	WorkerLostTimeout *metav1.Duration `json:"workerLostTimeout,omitempty"`
+  Dispatcher *MultiKueueWorkloadDispatcher `json:"multiKueueWorkloadDispatcher,omitempty"`
+}
+
+const(
+  DispatcherModeBuiltIn = "BuiltIn"
+  DispatcherModeSequential = "Sequential"
+)
+
+type MultiKueueWorkloadDispatcher struct {
+// DispatcherName defines the dispatcher responsible for selecting worker clusters to handle the workload.
+// - If specified, the workload will be handled by the named dispatcher.
+// - If not specified, the workload will be handled by the default (BuiltIn) dispatcher.
+  DispatcherName *string `json:"dispatcherName,omitempty"`
```

Comment:
For pluggable APIs use the domain prefix to indicate the plugin provider. 

For example look at definition of Job [ManagedBY API](https://github.com/kubernetes/kubernetes/blob/8458cfed43f96e05fac0bfa43fbec473c39d5689/pkg/apis/batch/types.go#L462-L470), example value being `kueue.x-k8s.io/multikueue`.

Maybe:  `kueue.x-k8s.io/multikueue-dispatcher-all-at-once` and `kueue.x-k8s.io/multikueue-dispatcher-sequential` (but we can discuss particular names in other comments)